composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": ">=5.4.8",
+        "php": "^5.5.9||^7.0.0",
         "symfony/event-dispatcher": "^2.3||^3.0",
         "symfony/options-resolver": "^2.3||^3.0",
-        "psr/http-message-implementation": "~1.0",
-        "php-http/adapter-implementation": "^0.1.0",
-        "php-http/discovery": "^0.1.1",
-        "php-http/message-decorator": "^0.1.0"
+        "php-http/client-implementation": "^1.0.0",
+        "php-http/discovery": "^0.6.0",
```

Comment:
That's what I would like to achieve: https://github.com/php-http/discovery/issues/33